### ISSUE TITLE
Provide an option to disable automatic bind of logs

### ISSUE THREAD

**Issue by yujiosaka** [ladjs/i18n#7]
> Note that we automatically bind `logDebugFn`, `logWarnFn`, and `logErrorFn` for i18n options to logger.debug, logger.warn, and logger.error respectively.

Because of this feature, especially when I run tests, following logs are printed in my console:

```
console.debug node_modules/i18n/i18n.js:1165
  will use /Users/yujiosaka/work/myapp/locales/ja.json
console.debug node_modules/i18n/i18n.js:1165
  read /Users/yujiosaka/work/myapp/locales/ja.json for locale: ja
console.debug node_modules/i18n/i18n.js:1165
  will use /Users/yujiosaka/work/myapp/locales/en.json
...
```

Because of these logs, it becomes difficult for me to locate other important messages.

In addition to that, I don't find this binding necessary.
Because the default logs are bound to [debug](https://www.npmjs.com/package/debug) module,
you can see their logs just by passing `DEBUG=i18n:*`.

So I suggest the following two options:

1. Provide an option to disable this feature
2. Do not bind logs in this module

I recommend the option 2. But either way, I can create an PR if you allow me.

**Comment by niftylettuce** [comment]
I would gladly accept a pull request @yujiosaka

**Comment by niftylettuce** [comment]
Please install v1.2.0+ of `@ladjs/i18n` as this has been fixed and released!

```sh
npm install @ladjs/i18n@latest
```

I have also published a deprecation notice for versions <1.2.0 which reads:

```sh
Please upgrade to v1.2.0+ as we have removed automatic binding of log functions to i18n (which polluted your console accidentally)
```
